lib/Search/FileSearch/FileSearcher.php

```diff
@@ -21,13 +21,29 @@ class FileSearcher extends TNTSearch {
 		'storage' => ''
 	];
 
+	public const SUPPORTED_LANGUAGES = [
+		'ar' => 'Arabic',
+		'cr' => 'Croatian',
+		'fr' => 'French',
+		'de' => 'German',
+		'en' => 'Porter',
+		'it' => 'Italian',
+		'lv' => 'Latvian',
+		'pl' => 'Polish',
+		'pt' => 'Portuguese',
+		'ru' => 'Russian',
+		'uk' => 'Ukrainian',
+	];
+
 	protected FileIndexer $indexer;
+	private string $language;
 
-	public function __construct() {
+	public function __construct(string $language = 'no') {
```

Comment:
```suggestion
	public function __construct(?string $language = null) {
```

Would prefer this to not be confused with no for Norwegian

Reply:
fixed, but we should to bypass `'No'` explicitly to `\TeamTNT\TNTSearch\Indexer\TNTIndexer::setLanguage`

Reply:
@juliushaertl can you re-check/approve? :pray: